src/course-unit/hooks.jsx

```diff
@@ -73,6 +75,15 @@ export const useCourseUnit = ({ courseId, blockId }) => {
     dispatch(createNewCourseXBlock(body, callback, blockId))
   );
 
+  const unitXBlockActions = {
+    handleDelete: (XBlockId) => {
+      dispatch(deleteUnitItemQuery(blockId, XBlockId));
+    },
+    handleDuplicate: (XBlockId) => {
+      dispatch(duplicateUnitItemQuery(blockId, XBlockId));
+    },
+  };
+
```

Comment:
If we rename `blockId` to `locationId`, we need to have unnecessary variable renaming because we use common `blockId` variable the same as in other cases